src/controllers/legacycontrollersettings.cpp

```diff
@@ -118,7 +120,21 @@ QWidget* LegacyControllerBooleanSetting::buildInputWidget(QWidget* pParent) {
         emit changed();
     });
 
-    return pCheckBox;
+    auto pLabelWidget = make_parented<QLabel>(pWidget);
```

Comment:
what is this change for?

Reply:
This change was requried as the `QCheckBox` built-in label doesn't support rich text. Happy to move that to an other PR is we are happy about the way it looks with rich text (that is the emphasis on button )
![image](https://github.com/mixxxdj/mixxx/assets/7086688/3159f795-03a9-4ce8-bc14-8e190e2a1fc8)